### PR TITLE
Add hotkeys (also to help)

### LINKED ISSUE #304: Not all tools have hotkeys to reach them
Since the help has now been ported, it becomes apparent that some tools have hotkeys to activate (and/or deactivate) them, while others don't. I don't see a good reason why this should be the case.

Here is a suggestion for hotkeys for already ported tools that don't have one so far. What do you think @wjwwood ?

Focus Camera: C   (F is taken by some commands)
Publish Point Tool: U (P is taken)
Measure Tool: N  (M is taken, close by...)

### PR DESCRIPTION
**Pull request by Martin-Idel-SI**
Closes #304 

Also added hotkeys to help description to make discovery easy. 
Note: This PR should probably only be merged when a corresponding PR is merged with ros-visualization/rviz

### CODE REVIEW

**Comment by greimela-si** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4754/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1622/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3942/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4814/)

**Comment by Martin-Idel-SI** [comment]
CI after rebase:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4788/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1650/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3971/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4843/)

**Comment by greimela-si** [comment]
CI after rebase:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4808/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1669/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3992/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4861/)

File: rviz_default_plugins/src/rviz_default_plugins/tools/focus/focus_tool.cpp [hunk 1]
```diff
@@ -73,6 +73,8 @@ FocusTool::~FocusTool() = default;
 
 void FocusTool::onInitialize()
 {
+  shortcut_key_ = 'c';
```
Comment (wjwwood) [inline rviz_default_plugins/src/rviz_default_plugins/tools/focus/focus_tool.cpp]: Should this be in the constructor like the other two?

File: rviz_default_plugins/src/rviz_default_plugins/tools/measure/measure_tool.cpp [hunk 2]
```diff
@@ -58,6 +58,8 @@ namespace tools
 MeasureTool::MeasureTool()
 : is_line_started_(false), length_(-1)
 {
+  shortcut_key_ = 'n';
```
Comment (wjwwood) [inline rviz_default_plugins/src/rviz_default_plugins/tools/measure/measure_tool.cpp:61]: Should all of these be part of the member initialization section? `: is_line_..., shurtcut_key_('n')`?

Comment (Martin-Idel-SI) [reply rviz_default_plugins/src/rviz_default_plugins/tools/measure/measure_tool.cpp:61]: The problem is that they are protected members of the `rviz_common::Tool` class, so I can't do that - I would have to provide another non-default constructor for Tool.
 
There is a comment for the Tool constructor saying that the `shortcut_key_` member should be initialized in the constructor.

**Comment by Martin-Idel-SI** [comment]
CI after rebase and fixes:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/5009/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1820/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/4166/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/5021/)

**Review by wjwwood — APPROVED** [review]

**Pull request merged**